module-4/assignments/activity-1_N01499969.md

```diff
@@ -0,0 +1,13 @@
+1. Target 2nd product in the list
+//product[2]
+![image info](../assignments/1.jpg)
+2. Target last product in the list
+(//product)[last()]
+![image info](../assignments/2.jpg)
+3. Target `sku` attribute of the first product
+ //product[1]/@sku
+ ![image info](../assignments/3.jpg)
+4. Target all products with manufacturer id `sjb-pet`
+ //products/product/manufacturer[@id='sjb-pet']
```

Comment:
this targets all manufacturers, not products. you need to go one level up after the check